build.gradle.kts

```diff
@@ -22,15 +22,16 @@ dependencies {
 
   implementation("org.flywaydb:flyway-core")
   implementation("io.hypersistence:hypersistence-utils-hibernate-60:3.3.2")
+  implementation("org.hibernate.orm:hibernate-community-dialects:6.2.3.Final")
```

Comment:
actually I think you'll be able to get away with
```
  implementation("org.hibernate.orm:hibernate-community-dialects")
```
which will then float on the version of hibernate